tests/jenkins/install_coreneuron.sh

```diff
@@ -18,36 +18,35 @@ else
     mkdir build_${CORENRN_TYPE} build_intel_${CORENRN_TYPE}
 fi
 
+# default partition is interactive. during night use production
+hour=`date +%H`
+weekday=`date +%u`
+if [ "$hour" -ge "19" ] || [ "$hour" -lt "8" ] || [ $weekday -gt 5 ]; then
+  export SALLOC_PARTITION="prod";
+else
+  export SALLOC_PARTITION="interactive"
+fi
+
 cd $WORKSPACE/build_${CORENRN_TYPE}
 
 echo "${CORENRN_TYPE} build"
 if [ "${CORENRN_TYPE}" = "GPU-non-unified" ]; then
     cmake \
-        -DCMAKE_C_FLAGS:STRING="-O2" \
-        -DCMAKE_CXX_FLAGS:STRING="-O2 -D_GLIBCXX_USE_CXX11_ABI=0 -DR123_USE_SSE=0" \
-        -DCOMPILE_LIBRARY_TYPE=STATIC  \
-        -DCUDA_HOST_COMPILER=`which gcc` \
-        -DCUDA_PROPAGATE_HOST_FLAGS=OFF \
         -DCORENRN_ENABLE_GPU=ON \
         -DCORENRN_ENABLE_CUDA_UNIFIED_MEMORY=OFF \
         -DCMAKE_INSTALL_PREFIX=$WORKSPACE/install_${CORENRN_TYPE}/ \
-        -DTEST_MPI_EXEC_BIN="srun;--exclusive;--account=proj16;--partition=interactive;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
-        -DTEST_EXEC_PREFIX="srun;-n;6;--exclusive;--account=proj16;--partition=interactive;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
+        -DTEST_MPI_EXEC_BIN="srun;--account=proj16;--partition=$SALLOC_PARTITION;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
+        -DTEST_EXEC_PREFIX="srun;-n;6;--account=proj16;--partition=$SALLOC_PARTITION;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
         -DAUTO_TEST_WITH_SLURM=OFF \
         -DAUTO_TEST_WITH_MPIEXEC=OFF \
         $WORKSPACE/
 elif [ "${CORENRN_TYPE}" = "GPU-unified" ]; then
     cmake \
-        -DCMAKE_C_FLAGS:STRING="-O2" \
-        -DCMAKE_CXX_FLAGS:STRING="-O2 -D_GLIBCXX_USE_CXX11_ABI=0 -DR123_USE_SSE=0" \
-        -DCOMPILE_LIBRARY_TYPE=STATIC  \
-        -DCUDA_HOST_COMPILER=`which gcc` \
-        -DCUDA_PROPAGATE_HOST_FLAGS=OFF \
         -DCORENRN_ENABLE_GPU=ON \
         -DCORENRN_ENABLE_CUDA_UNIFIED_MEMORY=ON \
         -DCMAKE_INSTALL_PREFIX=$WORKSPACE/install_${CORENRN_TYPE}/ \
-        -DTEST_MPI_EXEC_BIN="srun;--exclusive;--account=proj16;--partition=interactive;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
-        -DTEST_EXEC_PREFIX="srun;-n;6;--exclusive;--account=proj16;--partition=interactive;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
+        -DTEST_MPI_EXEC_BIN="srun;--account=proj16;--partition=$SALLOC_PARTITION;--constraint=volta;--gres=gpu:1;--mem;0;-t;00:05:00" \
```

Comment:
Shall we do the gpu tests with 2 or more gpus to test this feature?

Reply:
yeah thats good idea. I changed it.